app/screens/sidebar/sidebar.less

```diff
@@ -7,7 +7,7 @@
   background-color: @primary-contrast-color;
   color: @primary-color;
   position: absolute;
-  width: 400px;
+  width: 355px;
```

Comment:
With 320px only owner Helas expands to two lines so that might be a good option as well. But we can keep 355px for now and I'll ask the customer whether that's narrow enough.